__tests__/app.test.js

```diff
@@ -131,4 +131,72 @@ describe('GET api/articles', () => {
                 expect(articles[12].created_at).toBe('2020-01-07T14:08:00.000Z')
             })
     })
+});
+
+describe('GET api/articles/:article_id/comments', () => {
+    test('sending an API request to api/articles/:articles_id/comments returns a Status-200: responds with an object, with a key comments, which has the value of an array of comments relevant to given id - in this case, article id 1 should have 11 comments.', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                const comments = body.comments
+                expect(comments).toHaveLength(11)
+            })
+    })
+    test('sending an API request to api/articles/:articles_id/comments returns a Status-200: responds with an object, with a key comments, which has the value of an array of comments relevant to given id - in this case, article id 3 should have 2 comments. Each comment has the required properties', () => {
```

Comment:
another exaple - this could be shortened to: Each comment has the required properties